lib/store/index.js

```diff
@@ -7,12 +7,25 @@
 import { createStore, applyMiddleware } from 'redux';
 import reducer from './reducer';
 import thunk from 'redux-thunk';
+import createLogger from 'redux-logger';
+import config from '../../config';
+
+const middlewares = [
+  thunk,
+];
+
+if (config.flags.reduxLogger) {
```

Comment:
If logging redux whenever we are in Debug mode is our intention, we should be explicit about that here, it'sn ot 'configurable'.